src/app/flux/store.js

```diff
@@ -263,7 +198,6 @@ const Store = Object.assign(
           type: 'posts'
         }], applyMorePosts).then(() => {
           Store.emit('change', _state);
```

Comment:
Niggly one but I'm just remembering, before I added in the `getSocialSharesForPost` call this line was in the condensed arrow function form...
```
}], applyMorePosts).then(() => Store.emit('change', _state));
```
Might be nice to restore that :)